osidb/tests/test_endpoints.py

```diff
@@ -1720,7 +1720,6 @@ def test_flawreference_create(self, auth_client, embargo_access, test_api_uri):
         flaw = FlawFactory()
 
         flawreference_data = {
-            "flaw": str(flaw.uuid),
```

Comment:
I'm not sure what this change accomplishes, perhaps the bit that removes the flaw as required param is in another commit?

Reply:
This is in a separate commit with the message "Do not require flaw value in body, because it is already provided in URL".

Jakub [requested](https://github.com/RedHatProductSecurity/osidb/pull/265#discussion_r1250405741) that this change be done for FlawAcknowledgments.

Jitka [requested](https://github.com/RedHatProductSecurity/osidb/pull/265#discussion_r1263797140) other FlawReference fixes in this PR to mirror how FlawAcknowledgments are handled.

Jitka [requested](https://github.com/RedHatProductSecurity/osidb/pull/265#discussion_r1262218968) that this change is in a separate commit.

Reply:
I mean that this commit does not do what it says it does "Do not require flaw value in body, because it is already provided in URL", the commit simply modifies a test, does not change anything about the actual code, given the commit message I would imagine some sort of change to the serializer.

Furthermore, shouldn't passing the flaw id parameter return a 400 response? That might be worthy of a bug ticket in JIRA

Reply:
@Elkasitu The code change happened in https://github.com/RedHatProductSecurity/osidb/pull/256/commits/15c9664494e00bd2a49143c97f14e6f6a70f1e61 in `osidb/serializer.py`. At that time, I forgot to update the test. And maybe, I forgot to update even more tests. I will check it and create another PR with FlawReference fixes, hopefully the last one :)

I agree that the used commit name ("Do not require flaw value in body, because it is already provided in URL".) is a bit confusing.

Reply:
Aha, I meant "Change the test to not provide the flaw value to be inside body, because it is already provided in the URL." I saw the test as something that effectively places requirements on how stuff should work (test should pass). Would the revised commit message in the first sentence work?

Theoretically, yes, it should return a 400, but that would IMO require changes I'm not sure are short enough for us to want. The reason the extra "flaw" in body is not detected (and in fact, always ignored) is that the viewset's `get_serializer()` injects the `flaw` from URL into the body, overwriting any potential value already in the body. The fix to return a 400 would probably be to have an extra condition in `get_serializer()` to detect the extra `flaw` in body and return a 400 response. Shall I file the ticket?

Reply:
Removing the commit removes the bad message.